.github/workflows/test-integration.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
         runner-version: [
           "pangeo-forge-runner==0.9.1",
           "pangeo-forge-runner==0.9.2",
+          "git+https://github.com/pangeo-forge/pangeo-forge-runner.git@gcorradini/add_back_context_manager#egg=pangeo-forge-runner",
```

Comment:
will have to be updated after `pangeo-forge-runner==0.9.3` release (see "Release Steps" in PR conversation)